packages/graphql/src/translate/create-auth-and-params.ts

```diff
@@ -67,6 +67,10 @@ function createAuthPredicate({
     }
     const { jwt } = context;
 
+    if (!jwt) {
+        throw new Error("Can't generate auth predicate - no JWT in context");
```

Comment:
This condition is making the whole graphql schema unusable for a non-authed user if `jwt: {}` is not set in the context.
It may be good to update docs accordingly.

Reply:
Hey @mathix420. This doesn't sound like a feature! Could you provide something reproducible?

Reply:
Hello @danstarns! I will try to give you that today, but i'm quite busy right now so it might be for this weekend.

Reply:
Eventually work can wait 10 minutes :sweat_smile:
```typescript
const { Neo4jGraphQL } = require("@neo4j/graphql");
const { ApolloServer } = require("apollo-server");
const neo4j = require("neo4j-driver");
require('dotenv').config();

const AURA_ENDPOINT = process.env.NEO4J_URL;
const PASSWORD = process.env.NEO4J_PASS;
const USERNAME = 'neo4j';

const typeDefs = `
    type Video {
        id: ID! @id
        title: String!
        publisher: String!
    }

    extend type Video @auth(rules: [
        { where: { publisher: "$jwt.sub" } },
    ]),
`;

const driver = neo4j.driver(AURA_ENDPOINT, neo4j.auth.basic(USERNAME, PASSWORD),{
      disableLosslessIntegers: true,
});

const neoSchema = new Neo4jGraphQL({ typeDefs, driver });

const server = new ApolloServer({
    schema: neoSchema.schema,
    context: ({ req }) => ({ req }),
});

server.listen(4000).then(() => console.log("Online"));
```

```graphql
query listVideos {
    videos {
        id
        title
    }
}
```

```json
{
  "errors": [
    {
      "message": "Can't generate auth predicate - no JWT in context",
      ...
    }
  ],
  "data": null
}
```

I think the issue is comming from the `where` rule.

Reply:
@mathix420 Thank you 🙏 we will look into this.

Reply:
**EDIT:** The "Quick workaround" is actually not working at all :stuck_out_tongue: 
It just avoid to fall into the `!jwt` condition, but never checks the jwt, and do not load jwt from cookies/headers.